Bottom: 0661b1e6af0fde76e36f815c48515f3ef47f9044
Top:    10c0603e82f16fda92188aa5294c129cab457160
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-27 17:05:53 -0700

Refresh of rstream-bufsize

---

diff --git a/examples/rstream.c b/examples/rstream.c
index 6ff4f1c..df36e34 100644
--- a/examples/rstream.c
+++ b/examples/rstream.c
@@ -350,7 +350,7 @@ out:
 
 static void set_options(int rs)
 {
-	int val, ret;
+	int val;
 
 	if (buffer_size) {
 		rs_setsockopt(rs, SOL_SOCKET, SO_SNDBUF, (void *) &buffer_size,
